_opcodes/op22-bgee.html

```diff
@@ -0,0 +1,38 @@
+---
+n: 22
+name: "Stat: Cumulative Luck Bonus"
+param1: "Statistic Modifier"
+param2: "Type"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Applies the modifier value specified by the 'Statistic Modifier' field in the style specified by the 'Type' field. Luck affects the minimum damage caused by a weapon (up to the normal maximum damage of the weapon) and the attack roll modifier (up to a maximum of 20, excluding natural attack roll modifiers, e.g. high STR, invisibility etc) by 'Statistic Modifier' amount. 'Statistic Modifier' also affects the damge rolls of opponents (in feature blocks, not extension headers). The effect does not set <a href="../files/ids/bgee/state.htm#BIT26">STATE_LUCK</a>.
+<br/>Known values for 'Type' are:
+<ul>
+	<li>0 &longrightarrow; Cumulative Modifier: Luck = Luck + 'Statistic Modifier' value</li>
+	<li>1 &longrightarrow; Flat Value Modifier: Luck = 'Statistic Modifier' value</li>
+	<li>2 &longrightarrow; Percentage Modifier: Luck = (Luck * 'Statistic Modifier' value) / 100</li>
+</ul>
+
+{% capture note %}
+Effects of Luck:
+<ul>
+	<li>Modifies a creature's weapon damage die rolls (up/down to its maximum/minimum).</li>
+	<li>Modifies a creatures's attack roll [1d20] (up/down to its maximum/minimum).</li>
+	<li>Modifies non-weapon damage die rolls dealt to the creature (up/down to its maximum/minimum).</li>
+	<li>Modifies a creatures's initiative roll [1d6 - 4] for weapon attacks (up/down to its maximum/minimum).</li>
+	<li>Modifies a creatures's initiative roll [1d8 - 1] for movement (up/down to its maximum/minimum).</li>
```

Comment:
I've never heard of movement initiative. :s Which stat is this related to or where can more info be found?

Reply:
> I've never heard of movement initiative. :s Which stat is this related to or where can more info be found?

Added more info about that.
Should I also include a [link](https://forums.beamdog.com/discussion/comment/1055283/#Comment_1055283) to this post?

Reply:
Interesting, sounds like a pathfinder hack. This is more than enough, thanks!